src/modules/queries/constants.ts

```diff
@@ -25,6 +26,9 @@ export const SET_QUERY_LIMIT_REACHED = '@queries/SET_QUERY_LIMIT_REACHED';
 export const GET_ORGANIZATION_USAGE_LIMITS =
   '@queries/GET_ORGANIZATION_USAGE_LIMITS';
 export const CLONE_SAVED_QUERY = '@queries/CLONE_SAVED_QUERY';
+export const SET_EXTRACTION_CONFIRMATION =
+  '@queries/SET_EXTRACTION_CONFIRMATION';
+export const CANCEL_EXTRACTION = '@queries/CANCEL_EXTRACTION';
```

Comment:
Constant values ​​that are no longer associated with redux can be deleted

Reply:
Thanks! Fixed :)